File: app/src/main/java/com/example/dora/ARouterPath.java
package com.example.dora;

/**
 * 这是一张本游乐园全景地图，可以告诉你哪个地方怎么走。
 */
public interface ARouterPath {

    String GROUP_MAIN = "/Main";
    String ACTIVITY_INTRODUCE = GROUP_MAIN + "/IntroduceActivity";
    String ACTIVITY_MENU_LIST = GROUP_MAIN + "/MenuListActivity";
    String ACTIVITY_WEB_VIEW = GROUP_MAIN + "/WebViewActivity";
    String ACTIVITY_EXTRAS = GROUP_MAIN + "/ExtrasActivity";
    String ACTIVITY_EVENT_BUS = GROUP_MAIN + "/EventBusActivity";
    String ACTIVITY_CRASH = GROUP_MAIN + "/CrashActivity";
    String ACTIVITY_TIPS = GROUP_MAIN + "/TipsActivity";
    String ACTIVITY_ORM = GROUP_MAIN + "/OrmActivity";
    String ACTIVITY_RSA = GROUP_MAIN + "/RsaActivity";
    String ACTIVITY_DATA_CACHE = GROUP_MAIN + "/DataCacheActivity";
    String ACTIVITY_PROGRESS_VIEW = GROUP_MAIN + "/ProgressViewActivity";
    String ACTIVITY_TITLE_BAR = GROUP_MAIN + "/TitleBarActivity";
    String ACTIVITY_ALERT_DIALOG = GROUP_MAIN + "/AlertDialogActivity";
}

File: common/src/main/java/com/example/common/ARouterPath.kt
package com.example.common

/**
 * ARouter的路径。
 */
interface ARouterPath {
    companion object {
        const val GROUP_MAIN: String = "Main"
        const val GROUP_DORA: String = "Dora"
        const val GROUP_CACHE: String = "Cache"
        const val GROUP_VIEW: String = "View"
        const val ACTIVITY_MENU_LIST: String = "/$GROUP_MAIN/MenuListActivity"
        const val ACTIVITY_INTRODUCE: String = "/$GROUP_DORA/IntroduceActivity"
        const val ACTIVITY_WEB_VIEW: String = "/$GROUP_DORA/WebViewActivity"
        const val ACTIVITY_EXTRAS: String = "/$GROUP_DORA/ExtrasActivity"
        const val ACTIVITY_EVENT_BUS: String = "/$GROUP_DORA/EventBusActivity"
        const val ACTIVITY_CRASH: String = "/$GROUP_DORA/CrashActivity"
        const val ACTIVITY_TIPS: String = "/$GROUP_DORA/TipsActivity"
        const val ACTIVITY_FLOW_PAGE: String = "/$GROUP_DORA/FlowPageActivity"
        const val ACTIVITY_NET_DETECT: String = "/$GROUP_DORA/NetDetectActivity"
        const val ACTIVITY_VIEW_MODEL_SAMPLE_1: String =
            "/$GROUP_DORA/ViewModelSample1Activity"
        const val ACTIVITY_VIEW_MODEL_SAMPLE_2: String =
            "/$GROUP_DORA/ViewModelSample2Activity"
        const val ACTIVITY_RSA: String = "/$GROUP_DORA/RsaActivity"
        const val ACTIVITY_WEB3_PAY: String = "/$GROUP_DORA/Web3PayActivity"
        const val ACTIVITY_ORM: String = "/$GROUP_CACHE/OrmActivity"
//        const val ACTIVITY_DATA_CACHE: String = "/$GROUP_CACHE/DataCacheActivity"
        const val ACTIVITY_PROGRESS_VIEW: String = "/$GROUP_VIEW/ProgressViewActivity"
        const val ACTIVITY_TITLE_BAR: String = "/$GROUP_VIEW/TitleBarActivity"
        const val ACTIVITY_TOGGLE_BUTTON: String = "/$GROUP_VIEW/ToggleButtonActivity"
        const val ACTIVITY_TOAST: String = "/$GROUP_VIEW/TipsActivity"
        const val ACTIVITY_TAB_LAYOUT: String = "/$GROUP_VIEW/TabLayoutActivity"
        const val ACTIVITY_ALERT_DIALOG: String = "/$GROUP_VIEW/AlertDialogActivity"
        const val ACTIVITY_LOADING_DIALOG: String = "/$GROUP_VIEW/LoadingDialogActivity"
        const val ACTIVITY_BOTTOM_DIALOG: String = "/$GROUP_VIEW/BottomDialogActivity"
        const val ACTIVITY_RADIO_GROUP: String = "/$GROUP_VIEW/RadioGroupActivity"
    }
}
